src/stories/Autocomplete/Component/AdvancedParameters.stories.tsx

```diff
@@ -97,3 +98,21 @@ addComponentStoryDescription(
   `const args = ${stringifyWithDefaults(Debounce.args)}`,
   debounceDescription
 );
+
+export const Translations = ComponentTemplate.bind({});
+Translations.args = {
+  apiKey,
+  onSubmit,
+  advancedParameters: {
+    numTermsWithGroupSuggestions: 1,
+    numGroupsSuggestedPerTerm: 2,
+    translations: {
```

Comment:
This looks just like passing key strings that it doesn't make sense to call it translations.

What's the difference between this and passing a param called "inTextValue" for example

I know we discussed this a little earlier but I'd imagine translations to accept different languages and handle the logic for rendering the matching language for me not just taking in one language

Reply:
I believe it should at least accept 

{
   "en": {"in": "inside"},
    "it": {"in": "..."}
}

Reply:
@mocca102 I guess that's what we agreed on earlier 😅 
@esezen correct me if I'm wrong; We agreed that we'd take in an object with direct translations of our translatable words (in our case just `in` for now), rather than taking in the whole translations JSON object and the current language (2 parameters).

There's no point of taking in translations for all languages since only one language will be active at any certain point of time. 

I'll bring up Enes's example once again.

```
const lang = "en" // Current language known by consumer at all times
const locales = import('locales.json');  //   Consumer's JSON file that has all translations
//   "en": {
//      "in": "inside"
//    },
//    "fr": {
//      "in": "..."
//    }

<CioAutocomplete
  language="en"
  translations={locales}
/>
```

So now rather than we get this entire json file and language consumers can pass us the snippet of this json for the current language and that's it:

```
const lang = "en" // Current language known by consumer at all times
const locales = import('locales.json');  //   Consumer's JSON file that has all translations
//   "en": {
//      "in": "inside"
//    },
//    "fr": {
//      "in": "..."
//    }

<CioAutocomplete
  translations={locales[lang]}
/>
```

> What's the difference between this and passing a param called "inTextValue" for example?

Well, since we only have a single word to be translated there's no much of a difference, but later on if we added more words then the object structure will come in handy!

Reply:
@HHHindawy How did you explain my explanation better than I did? 

@mocca102 I thought that's what we agreed on but I'm open to discussing it further in one of the pairing sessions